doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -403,60 +417,209 @@ Response code:
+
+<table style="width: 100%; margin: 12px 0 0 0;">
+<tr><td colspan="2"><strong>Required Parameters</strong></td></tr>
+<tr><td colspan="2">none</td></tr>
+<tr><td colspan="2"><strong>Optional Parameters</strong></td></tr>
+<tr>
+  <td style="padding-right: 40px; width: 140px;">attrs</td>
+  <td>Comma-separated subset of the attributes to be removed. If this parameter is not
+  included, the entity itself is removed.</td>
+</tr>
+</table>
+
+Response:
+
+* Successful operation uses 204 No Content
+* Error use a non-200 and (optionally) an error payload. See subsection on Error Responses for more detail.
+
++ Parameters
+    + entityId: Boe_Idearium (required, string) - Entity ID
+    + attrs: temperature,humidity (optional, string) - Attributes to be removed. Coma separated list.
+
++ Response 204
+
+## ContextEntity by type and ID [/v2/entities/type/{entityType}/id/{entityId}]
+
+### Retrieve entity [GET /v2/entities/type/{entityType}/id/{entityId}?{attrs}]
+
+<table style="width: 100%; margin: 12px 0 0 0;">
+<tr><td colspan="2"><strong>Required Parameters</strong></td></tr>
+<tr><td colspan="2">none</td></tr>
+<tr><td colspan="2"><strong>Optional Parameters</strong></td></tr>
+<tr>
+  <td style="padding-right: 40px; width: 140px;">attrs</td>
+  <td>Comma-separated subset of the attributes to be included in the response. If this parameter is not
+  included, all the attributes are retrieved.</td>
+</tr>
+</table>
+
+The response is an object representing the entity identified by the type and ID. The object
+follows the JSON entity representation format (described in a section above).
+
+This operation must return only one entity element, but it may happen that there are more
+than one entity with the same ID and type (e.g. entities with same ID and type in different
+service paths). In those cases  an error mesage is returned, specifying in the description the
+URL that could be used to get the list of conflicting entities.
+
+Response:
+
+* Successful operation uses 20O OK
+* Error use a non-200 and (optionally) an error payload. See subsection on Error Responses for more detail.
 
 + Parameters
-    + entityId: 'Bcn_Welt' (required, string) - Entity ID
+    + entityId: Boe_Idearium (required, string) - Entity ID
+    + entityType: Room (required, string) - Entity Type
     + attrs: temperature, humidity (optional, string) - Attributes to be retrieved. Coma separated list.
 
 + Response 200 (application/json)
 
         {
+            "type": "Room",
+            "id": "Bcn_Welt",
             "temperature": 21.7,
             "humidity": 60,
+            "location": {
+                "value":  "41.3763726, 2.1864475",
+                "type": "geo:point",
+                "crs": "WGS84"
+            }
         }
 
-### Get entity attributes. Not present [GET /v2/entities/{entityId}{?attrs}]
+### Update or append entity attributes [POST /v2/entities/type/{entityType}/id/{entityId}?{op}]
```

Comment:
You mean in general to all the operations which use `/type/<type>/id/<id>" ? Could make sense...